_docs-sources/faq/terragrunt/index.md

```diff
@@ -0,0 +1,15 @@
+# Terragrunt FAQ
+
+## General questions
+
+- [Passing variables between Terragrunt and Terraform](https://github.com/gruntwork-io/knowledge-base/discussions/137)
+- [Mocking large sets of outputs with Terragrunt](https://github.com/gruntwork-io/knowledge-base/discussions/109)
```

Comment:
```suggestion
- [How do I pass variables between Terragrunt and Terraform?](https://github.com/gruntwork-io/knowledge-base/discussions/137)
- [Can I mock large sets of outputs with Terragrunt?](https://github.com/gruntwork-io/knowledge-base/discussions/109)
```

Reply:
I happened to make a suggestion here, though this comment applies more broadly. I think it would be nice to phrase _everything_ in the form of a question, for consistency.

Reply:
Roger - carried this throughout the PR in 4de5e1d5f18a406f1e8c01d4b426765e415df8bd, and re-phrased everything as a question, with the exception of 2 errors which are under an error header on the EKS page, since I believe that will assist them being found via exact-match searching.